### PR TITLE
tBTC activity history list

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #475
Closes: #471 

This PR updates the activity history item styles and renders a link that redirects to the deposit details page on item click.

<details>
  <summary>Screenshots</summary>
  
obraz

</details>

### CODE REVIEW

File: src/pages/tBTC/Bridge/TransactionHistory/index.tsx [hunk 1]
```diff
@@ -54,14 +59,54 @@ export const TransactionHistoryCard: FC<ComponentProps<typeof Card>> = ({
...
+      <TbtcActionBadge status={status} />
+    </ActivityItemWrapper>
+  )
+}
+
+const renderActivityItem = (item: BridgeTxHistory) => (
+  <ActivityItem key={item.depositKey} {...item} />
+)
+
+export const TransactionHistory: FC<{
```
Comment (michalsmiarowski) [inline src/pages/tBTC/Bridge/TransactionHistory/index.tsx:108]: Do you mind changing the component to `TransactionActivity` (and also this file to `TransactionActivity/index.tsx`)? Of course are related components to it too (like `TransactionHistoryCard` and `TransactionHistoryTabel`.

In our current designs Transaction History is now something else:
image

Comment (michalsmiarowski) [reply src/pages/tBTC/Bridge/TransactionHistory/index.tsx:108]: Optionally we could also change the `bridgeTxHistory` method  in `threshold-ts` lib to `bridgeTxActivity` and related `BridgeTxHisotry` interface to` BridgeTxActivity` if you wish.

File: src/pages/tBTC/Bridge/TransactionHistory/index.tsx [hunk 2]
```diff
@@ -2,24 +2,27 @@ import { ComponentProps, FC } from "react"
...
+import {
+  BridgeHistoryStatus,
+  BridgeTxHistory,
+} from "../../../../threshold-ts/tbtc"
 import emptyHistoryImageSrcDark from "../../../../static/images/tBTC-bridge-no-history-dark.svg"
 import emptyHistoryImageSrcLight from "../../../../static/images/tBTC-bridge-no-history-light.svg"
 import ViewInBlockExplorer from "../../../../components/ViewInBlockExplorer"
 import { ExplorerDataType } from "../../../../utils/createEtherscanLink"
 import { InlineTokenBalance } from "../../../../components/TokenBalance"
+import Link from "../../../../components/Link"
```
Comment (michalsmiarowski) [inline src/pages/tBTC/Bridge/TransactionHistory/index.tsx:26]: Non-blocking:

I also noticed that the Transaction History doesn't look good on smaller screen (but not that bad either, it's readable):
image


This one is more related to #475, but it's already merged so if it's easy to center that out for lower resolution then you can address it in this PR. If not, then let's addres it in the future.

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left some minor comments but overall looks very good. I would like test it out also after you merge the newest changes from `main`, because right now I have if deposit is minted I have to wait 20 seconds to see it (but it was fixed here: https://github.com/threshold-network/token-dashboard/pull/475#discussion_r1138549645)

File: src/pages/tBTC/Bridge/TransactionHistory/index.tsx (hunk 1 above)
Comment (r-czajkowski) [reply src/pages/tBTC/Bridge/TransactionHistory/index.tsx:108]: > Optionally we could also change the `bridgeTxHistory` method in `threshold-ts` lib to `bridgeTxActivity` and related `BridgeTxHisotry` interface to` BridgeTxActivity` if you wish.

Now I'm wondering if we should keep this method in the tBTC interface... 🤔 I think it's too specific for our T dapp to keep it in the tBTC interface. But I'll update and we can revisit it in the future.

File: src/pages/tBTC/Bridge/index.tsx [hunk 3]
```diff
@@ -29,9 +26,9 @@ const TBTCBridge: PageComponent = (props) => {
   const { openModal } = useModal()
   const { hasUserResponded } = useTBTCTerms()
   const dispatch = useAppDispatch()
-  const bridgeTransactionHistory = useAppSelector(selectBridgeTransacionHistory)
+  const bridgeActivity = useAppSelector(selectBridgeActivity)
   const isBridgeTxHistoryFetching = useAppSelector(
```
Comment (michalsmiarowski) [inline src/pages/tBTC/Bridge/index.tsx:30]: We should rename it to `isBridgeActivityFetching`

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]

File: src/pages/tBTC/Bridge/index.tsx (hunk 3 above)
Comment (r-czajkowski) [reply src/pages/tBTC/Bridge/index.tsx]: ad2692f

File: src/pages/tBTC/Bridge/index.tsx [hunk 4]
```diff
@@ -61,17 +58,17 @@ const TBTCBridge: PageComponent = (props) => {
       </Box>
       <Box gridArea="aside">
         <TbtcBalanceCard gridArea="balance-card" mb="5" />
-        <TransactionHistoryCard>
+        <BridgeActivityCard>
           {isBridgeTxHistoryFetching ? (
```
[the remark this answers is not in the document]
Comment (r-czajkowski) [reply src/pages/tBTC/Bridge/index.tsx]: ad2692f

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Pull request merged**